cmd/crc/cmd/start.go

```diff
@@ -248,9 +248,10 @@ Use the 'oc' command line interface:
   {{ .CommandLinePrefix }} {{ .EvalCommandLine }}
   {{ .CommandLinePrefix }} oc login -u {{ .ClusterConfig.DeveloperCredentials.Username }} {{ .ClusterConfig.URL }}
 `
-	startTemplateForPodman = `podman runtime is now running.
+	startTemplateForPodman = `podman runtime is currently configured in rootless mode. If your containers
+require root permissions, or if you run into compatibility
+issues with non-podman clients, you can switch using the following commands:
 
-Use the 'podman' command line interface:
   {{ .CommandLinePrefix }} {{ .EvalCommandLine }}
   {{ .CommandLinePrefix }} {{ .PodmanRemote }} COMMAND
```

Comment:
By default, we don't need to do anything  because we configured a podman connection, but to change to root, we don't document how to change the podman connection as described in the commit log, but we document something entirely different? How do I switch back to the rootless mode (I mean, if I don't know how this works at all, ,and only followed the instructions from this template?)